docs/api/api.md

```diff
@@ -174,42 +174,44 @@ documents](#project-documents).
+| ID                  | Integer      | Read Only          | The ID for the document                                                                                                                                          |
+| access              | String       | Default: `private` | The [access level](#access-levels) for the document                                                                                                              |
+| asset_url           | String       | Read Only          | The base URL to load this document's [static assets](#static-assets) from                                                                                        |
+| canonical_url       | URL          | Read Only          | The canonical URL to view this document                                                                                                                          |
+| created_at          | Date Time    | Read Only          | Time stamp when this document was created                                                                                                                        |
+| data                | JSON         | Not Required       | [Custom metadata](#data)                                                                                                                                         |
+| description         | String       | Not Required       | A brief description of the document                                                                                                                              |
+| edit_access         | Bool         | Read Only          | Does the current user have edit access to this document                                                                                                          |
+| file_url            | URL          | Create Only        | A URL to a publicly accessible document for the [URL Upload Flow](#url-upload-flow)                                                                              |
+| force_ocr           | Bool         | Create Only        | Force OCR even if the PDF contains embedded text - only include if `file_url` is set, otherwise should set `force_ocr` on the call to the processing endpoint    |
+| language            | String       | Default: `eng`     | The [language](#languages) the document is in                                                                                                                    |
+| organization        | Integer      | Read Only          | The ID for the [organization](#organizations) this document belongs to                                                                                           |
+| original_extension  | String       | Default: `pdf`     | The original file extension of the document you are seeking to upload using upload(), if none is provided it is assumed to be a PDF.                             |
```

Comment:
`upoad()` is a method in the python library, which is a wrapper around this API.  This API documentation should be neutral, allowing people to call it directly via any HTTP library or to implement there own wrapper in a lanaguage of choice.  Would probably be good to cross reference the `[supported file type](#supported-file-types)` here as well.